tests/simple/conversion.wast

```diff
@@ -24,4 +24,24 @@
 (i64.extend_i32_s (i32.const 15))
 #assertTopStack < i64 > 15 "folded extend sig"
 
+(module
```

Comment:
Is this a conformance test? Maybe better to just give it its own file, then?

Reply:
in conformance tests float and int conversions are put together, but now float related conformance tests are put separate in a folder called float, so I prefer to keep this for now.

Reply:
Ah, I see.